packages/node-sdk/src/client.ts

```diff
@@ -271,7 +277,12 @@ export class BucketClient {
 
       this._config.logger?.debug(`get request to "${url}"`, response);
 
-      if (!isObject(response.body) || response.body.success !== true) {
+      if (
+        !isObject(response.body) ||
+        response.body.success !== true ||
+        response.status < 200 ||
+        response.status >= 400
```

Comment:
I think we can just use `response.ok` instead of these last two

Reply:
Makes sense, would need to expose it.

Reply:
expose it?